### DISCUSSION TITLE
Grain boundaries shift when using the 'boundary, tight'

### DISCUSSION THREAD

**Discussion by TianchenW** [mtex-toolbox/mtex#1935] [Ask Anything]
Hi everyone,

I am a beginner and started using the MTEX to process my EBSD data recently. Because I need to collect all the orientation data through my whole sample thickness, a large area scan is necessary and there are a lot of noise points surround the sample which are from the sample mount. By using the  _'boundary' ,'tight'_  can help me to get a better sample outline, but it also gives me the grain boundary offset like that in the figure. The strange part is they can line up better in the left region than the middle and the right. And what I can see is there is another layer of grains is added on the top which is inaccurate. Could anyone come up with an idea about how to fix this issue to get sample outlines and the grain boundaries better?

Thanks a lot for helping in advance!

Kimi

![BoundaryTightIssue](https://github.com/mtex-toolbox/mtex/assets/93149052/37d05035-d9c7-475f-a0ba-635337a97335)

**Comment by ralfHielscher** [comment]
The offset should not be there. Which MTEX version are you using?

**Reply by TianchenW** [reply]
MTEX 5.8.1

**Comment by ralfHielscher** [comment]
Would it be possible for you to check it with MTEX 5.10?

Ralf
